condor/condor_meter

```diff
@@ -160,9 +163,10 @@ def get_num_procs(job_ad):
     If none of the above are set or do not evaluate to an integer, return 0
     """
     procs = 0
-    for attr in ['RequestCpus', 'MATCH_EXP_JOB_GLIDEIN_Cpus', 'MachineAttrCpus0']:
+    for attr in PROC_ATTRS:
```

Comment:
+1 for this (checking the attrs in order and returning after the first match_ - i was going to suggest you do something similar

Reply:
Yeah, the original way felt dirty. If there's nothing else, can you approve the PR?